controllers/util/tests/tests.go

```diff
@@ -0,0 +1,170 @@
+/**
+ * Copyright 2019 IBM Corp.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package tests
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"strings"
+
+	csiv1 "github.com/IBM/ibm-block-csi-operator/api/v1"
+	"github.com/IBM/ibm-block-csi-operator/pkg/config"
+	"gopkg.in/yaml.v2"
+	"k8s.io/apimachinery/pkg/types"
+)
+
+var (
+	relativeCrPath = "../config/samples/csi.ibm.com_v1_ibmblockcsi_cr.yaml"
```

Comment:
could we unify it with [this](https://github.com/IBM/ibm-block-csi-operator/blob/d013f44a5f04475d55b64b718c0a5fb1919a946a/build/Dockerfile.operator#L53) somehow?

Reply:
we can but I think it will be overkill since we already have this value in multiple places and the solution that I thought is not so easy to do, we have today good IDE that we can manage it pretty easy in the future

Reply:
1. where else do we have this value?
2. I don't think it's easy, even with a good IDE, there is no "rename" button for string values
3. what is the solution you had in mind? I thought about some simple [configuration file](https://github.com/IBM/ibm-block-csi-driver/blob/99463e0043bf14f7fa9df35c65bbe8c40a668910/common/config.yaml)

Reply:
1. [here](https://github.com/IBM/ibm-block-csi-operator/blob/d013f44a5f04475d55b64b718c0a5fb1919a946a/build/ci/Dockerfile.unittest#L20) and [here](https://github.com/IBM/ibm-block-csi-operator/blob/d013f44a5f04475d55b64b718c0a5fb1919a946a/build/Dockerfile.operator#L53)
2. in Vscode you can rename a string in all the files in a specific directory and I am sure that it can be done by other IDEs
3. that was basically what I thought, but you need to also write code in go and in bash that knows how to get it and I thought it harder than using the IDE, especially if we already have this gap, I can open a ticket for it if you want

Reply:
then please open a ticket for it

Reply:
done 
CSI-4034